01week/datatypes.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+//Write a JavaScript program to display the current day and time.
+const year = new Date().getFullYear();
+const month = new Date().getMonth();
+const day = new Date().getDate();
+const hour = new Date().getHours();
+const minute = new Date().getMinutes();
+
+console.log(`The day of our lord: ${month}/${day}/${year}`)
+console.log(`The time of our life: ${hour}:${minute}`)
+
+//this is called template literal using backticks
+
+console.log("It's " + new Date().getMonth() + "/" + new Date().getDate() + "/" + new Date().getFullYear());
+
+//Write a JavaScript program to convert a number to a string.
+
+const stringMe = 15;
+const strung = stringMe.toString();
+console.log(strung + ' is a ' + typeof strung);
+
+//Write a JavaScript program to convert a string to the number.
+
+const numMe = 25;
+const numbed = parseInt(numMe, 10);
```

Comment:
This is a variable that stores the number 25. I need a function that converts ANY string into a number and converts it.